modules/blockuser.cpp

```diff
@@ -24,6 +24,7 @@ using std::vector;
 class CBlockUser : public CModule {
   public:
     MODCONSTRUCTOR(CBlockUser) {
+        //Creating all commands
```

Comment:
I don't see how this comment helps. Name of `AddCommand` function already contains what it does, no?